react-accordion-component/src/accordion.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+
+class Accordion extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { current: null };
+    this.topics = [];
+    this.handleClick = this.handleClick.bind(this);
+  }
+
+  handleClick(event) {
+    if (event.target.matches('.topic-button')) {
+      if (this.state.current === event.target.topicId) {
+        this.setState({ current: null });
+      } else {
+        this.setState({ current: event.target.topicId });
+      }
+    }
+  }
+
+  render() {
+    this.topics = this.props.topics.map(topic => {
```

Comment:
It's not really hurting anything but using a `const` here would be more appropriate:

```JSX
const topics = ...

Reply:
@scbowler gotcha I had this code initially in my constructor but ran into the problem of needing queryselectors so I copied it down from up there. I also have this in my carousel code so i'll keep in mind to just use const in the future instead of assigning it as a property of the component